setup.py

```diff
@@ -1,28 +1,10 @@
 from setuptools import find_packages, setup
 
 # Runtime requirements.
-aws_cdk_version = "1.203.0"
-aws_cdk_reqs = [
-    "core",
-    "aws-lambda",
-    "aws-lambda-python",
-    "aws-secretsmanager",
-    "aws-sqs",
-    "aws-ec2",
-    "aws-rds",
-    "aws-stepfunctions",
-    "aws-stepfunctions-tasks",
-    "aws-events",
-    "aws-events-targets",
-    "aws-apigateway",
-    "aws-ssm",
-    "aws-s3",
-    "aws-lambda-event-sources",
-    "aws-logs",
-]
-
+aws_cdk_version = "2.162.1"
 inst_reqs = [
-    *[f"aws_cdk.{x}=={aws_cdk_version}" for x in aws_cdk_reqs],
+    f"aws-cdk-lib=={aws_cdk_version}",
+    f"aws-cdk.aws-lambda-python-alpha=={aws_cdk_version}a0",
```

Comment:
CDK v2 only has 1 main package but splits anything in "alpha" or "beta" off into separate packages. We're using `PythonFunction` from `aws-cdk.aws-lambda-python-alpha` so we need this additional package